submissions/TaniaBondarenko/friends-app/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Friends App</title>
+    <link href="styles.css" type="text/css" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Courgette&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body>
+    <div class="main">
+      <div class="sidebar">
+        <div class="sort">
+          <div class="byName">
+            <div class="text">SORT BY NAME</div>
+            <button class="ascendent_by_name" type="button"></button>
+            <button class="descendent_by_name" type="button"></button>
+          </div>
+        </div>
+        <div class="sort">
+          <div class="byAge">
+            <div class="text">SORT BY AGE</div>
+            <button class="ascendent_by_age" type="button"></button>
+            <button class="descendent_by_age" type="button"></button>
+          </div>
+        </div>
```

Comment:
Also can be a form or fieldset with inputs

Reply:
Got it

Reply:
Got it